packages/exenum/exenum.0.86/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Didier Le Botlan <github.lebotlan@dfgh.net>"
+authors: "Didier Le Botlan <github.lebotlan@dfgh.net>"
+homepage: "https://github.com/lebotlan/ocaml-exenum"
+bug-reports: "https://github.com/lebotlan/ocaml-exenum/issues"
+license: "MIT License"
+dev-repo: "git://github.com/lebotlan/ocaml-exenum.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: ["dune" "zarith"]
```

Comment:
Could you flag `dune` as a `{build}` dependency?

Reply:
Sure! (It would be worth adding it to the dune manual.)
Besides, the opam manual lacks an example. Is the following correct?
depends: ["dune" {build} "zarith"]

Reply:
> Is the following correct? depends: ["dune" {build} "zarith"]

Yes, though a more common/readable way would be to indent each dependencies as follow:
```
depends: [
  "dune" {build}
  "zarith"
]
```

Reply:
Thanks for your advice. I have opam-published the package again.